src/utils/fetchUtils.ts

```diff
@@ -28,20 +49,25 @@ export class FetchResponse extends Response {
   }
 
   static setUrl(url: string | undefined, response: Response): void {
-    if (!url) {
+    if (url == null || url === 'about:') {
```

Comment:
`about:` requests must keep `response.url` value equal to `''`:

<img width="678" alt="Screenshot 2025-04-04 at 11 42 57" src="https://github.com/user-attachments/assets/6f474ffa-7b2f-4577-9786-2b3170d74697" />